app/models/api_request.rb

```diff
@@ -21,14 +31,30 @@ def host
     raise "Please define a host method in #{self.class}"
   end
 
+  def page_limit
```

Comment:
[suggestion] 
having this conditional logic feels like a smell to me. it would be more OO to pull the logic into Api::Hses (or Api::Hses::Issues, whichever feels more appropriate) and have these methods return `""` so that they will be more configurable in different endpoints/systems.

other options:
* default_params could be just ["username=#{@username}&types=1"] and more default_params could be defined within the individual systems.
* if we are using page_limit and page_offset for all systems define it further upstream so we don't have to use the conditional